src/examples/Capi.cpp

```diff
@@ -269,7 +269,7 @@ void resize_cb(void *data, Evas *e, Evas_Object *obj, void *event_info)
 int main(int argc, char **argv)
 {
     elm_init(argc, argv);
-    tvg_engine_init(Tvg_Engine(TVG_ENGINE_SW | TVG_ENGINE_GL), 0);
```

Comment:
@mihashco  @mgrudzinska btw any issues at this? our thorvg capi example has no problems.

Reply:
@hermet because we did the explicit casting:
``` tvg_engine_init(Tvg_Engine(TVG_ENGINE_SW | TVG_ENGINE_GL), 0);```

Reply:
we can not call this funct like this ``` tvg_engine_init(TVG_ENGINE_SW | TVG_ENGINE_GL, 0);```
do you think is that ok that the user has to remember about the casting ?

Reply:
Well.. though it's not standard, I don't much see the trouble here. They can easily understand & apply the casting when it's necessary, otherwise they will copy from our examples. If you think this is a matter, then bit operation for the enum is another issue as well. So... please choose, leave it with good example code or revert your old patch: https://github.com/Samsung/thorvg/pull/914/files  :)

Reply:
@hermet if it's ok with you I think we can keep it as it is now.